scala-core-oop/src/main/scala/com/baeldung/scala/classvsobject/ClassExamples.scala

```diff
@@ -0,0 +1,68 @@
+package com.baeldung.scala.classvsobject
+
+object ClassExamples {
+
+  class Car (val manufacturer: String, var brand: String = "ty", var model: String) {
+    var speed: Double = 0;
+    var gear: Any = 0;
+    var isOn: Boolean = false;
+
+    def start(key_type: String): String = {
```

Comment:
Maybe it is intentional, but we seem to have a mixture of camel case and non-camel case for variable names throughout the code?